src/components/component-base/hole/hole.spec.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Hole from './index';
+import $ from 'teaspoon';
+
+describe('hole', function () {
+  it('should render class with __hole element', function () {
+    let result = $(<Hole className="test"/>).render();
```

Comment:
This is a `teaspoon` library. It mimics jquery for React components.
To be used in tests.